addons/point_of_sale/static/src/app/utils/money_details_popup/money_details_popup.xml

```diff
@@ -12,9 +12,9 @@
                             <div class="money-details-value d-flex align-items-center w-100 my-1 mx-auto" t-on-input="updateMoneyDetailsAmount">
                                 <input class="pos-input form-control w-50 text-end" t-att-id="moneyValue" type="number" t-model.number="state.moneyDetails[moneyValue]" t-on-focus="ev=>ev.target.select()"/>
                                 <label class="oe_link_icon w-25 text-end" t-att-for="moneyValue">
-                                    <t t-if="currency.position === 'before'"><t t-esc="currency.symbol"/>&nbsp;</t>
-                                    <t t-esc="moneyValue"/>
-                                    <t t-if="currency.position === 'after'">&nbsp;<t t-esc="currency.symbol"/></t>
+                                    <t t-if="currency.position === 'before'" t-esc="currency.symbol"/>
+                                    <t class="mx-1" t-esc="moneyValue"/>
```

Comment:
Sorry for missing this yesterday.
```suggestion
                                    <span class="mx-1" t-esc="moneyValue" />
```

It should be an html element so that the css utility class works.